.betterer.results

```diff
@@ -2772,11 +2772,6 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
     ],
-    "public/app/core/mod_defs.d.ts:5381": [
```

Comment:
@toddtreece if we sync main will this file go away?

If not, maybe manually remove it?